src/xcode/ENA/ENA/Source/Models/Metadata/Test Result Metadata/TestResultMetadata.swift

```diff
@@ -67,4 +82,29 @@ struct TestResultMetadata: Codable {
 	
 	// We need a copy of the token to compare it everytime we fetch a testResult to make sure it is a result for the QRCode test and not a TAN test submission
 	let testRegistrationToken: String
+
+	let testType: TestType
+
+	var protobuf: SAP_Internal_Ppdd_PPATestResult? {
+		switch (testType, testResult) {
+		case (.pcr, .pending):
+			return .testResultPending
+		case (.pcr, .negative):
+			return .testResultNegative
+		case (.pcr, .positive):
+			return .testResultPositive
+		case (.pcr, .expired), (.pcr, .invalid):
```

Comment:
Is invalid not mapped here on purpose? Seems like we have it in `SAP_Internal_Ppdd_PPATestResult` for pcr and antigen tests

Reply:
Yes, on purpose. I was wondering myself and needed to verify this with max.